src/components/Footer/index.jsx

```diff
@@ -0,0 +1,135 @@
+import React, { Fragment } from 'react';
+import styles from './styles.css';
+import boleto from './icons/boleto.svg';
+import elo from './icons/elo.svg';
+import mastercard from './icons/mastercard.svg';
+import paypal from './icons/paypal.svg';
+import visa from './icons/visa.svg';
+import telephone from './icons/telephone.svg';
+import email from './icons/email.svg';
+
+const footerLinks = [
```

Comment:
To make a generic `Footer` I suggest you pass these link to outside the component and use as children.

With this current implementation, we cannot use this `Footer` component in the `Checkout` page, per example.